<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>singleplay</class>
 <widget class="QWidget" name="singleplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Single</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>570</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;1훈화양연화 R&quot;;
color: rgb(237, 125, 49);</string>
   </property>
   <property name="text">
    <string notr="true">시작</string>
   </property>
  </widget>
  <widget class="QLabel" name="imagelabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>400</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 213, 213);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>570</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;1훈화양연화 R&quot;;
color: rgb(237, 125, 49);</string>
   </property>
   <property name="text">
    <string notr="true">시작</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>650</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;1훈화양연화 R&quot;;
color: rgb(237, 125, 49);</string>
   </property>
   <property name="text">
    <string notr="true">시작</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>650</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;1훈화양연화 R&quot;;
color: rgb(237, 125, 49);</string>
   </property>
   <property name="text">
    <string notr="true">시작</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
